src/engine/client/key_binding.cpp

```diff
@@ -75,7 +75,6 @@ static const struct
 	{ "super", 5, 32, K_SUPER },
 	{ "compose", 6, 64, K_COMPOSE },
 	{ "menu", 7, 128, K_MENU },
-	{ "" }
```

Comment:
This can't be removed as the zero array element is used in the loop condition `modifierKeys[ i ].bit`. Instead, you could change it from `{ "" }` to `{}`.

Reply:
My bad, I didn't realize it.